commit 1c15b334ca34dd22a19e1c595908fc860523be64
Author: Shuo Liu <shuo.a.liu@intel.com>
Date:   Thu Mar 11 09:52:06 2021 +0800

    virt: acrn: Fix document of acrn_msi_inject()
    
    This fixes below build warning with extra build checks.
    
    $ make W=1
    
    ../drivers/virt/acrn/vm.c:105: warning: expecting prototype for
    acrn_inject_msi(). Prototype was for acrn_msi_inject() instead
    
    Fixes: c7cf8d27244f ("virt: acrn: Introduce interrupt injection interfaces")
    Signed-off-by: Shuo Liu <shuo.a.liu@intel.com>
    Link: https://lore.kernel.org/r/20210311015206.19715-1-shuo.a.liu@intel.com
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/virt/acrn/vm.c b/drivers/virt/acrn/vm.c
index 7804a2492ad7..0d002a355a93 100644
--- a/drivers/virt/acrn/vm.c
+++ b/drivers/virt/acrn/vm.c
@@ -94,7 +94,7 @@ int acrn_vm_destroy(struct acrn_vm *vm)
 }
 
 /**
- * acrn_inject_msi() - Inject a MSI interrupt into a User VM
+ * acrn_msi_inject() - Inject a MSI interrupt into a User VM
  * @vm:		User VM
  * @msi_addr:	The MSI address
  * @msi_data:	The MSI data